spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -806,48 +875,84 @@ function timeoutChannelUpgrade(
     proofHeight: Height,
 ) {
     // current channel must be in INITUPGRADE
-    currentChannel = provableStore.get(channelPath(portIdentifier, channelIdentifier))
-    abortTransactionUnles(currentChannel.state == INITUPGRADE)
-
-    upgradeTimeout = provableStore.get(timeoutPath(portIdentifier, channelIdentifier))
```

Comment:
This was not correct, since the timeout is a field of `Upgrade`, so I changed it to retrieve the upgrade first from the upgrade path and then access the `timeout` field.